sw/device/silicon_creator/lib/drivers/otbn.h

```diff
@@ -33,11 +33,11 @@ enum {
  * Example:
  * ```
  *  otbn_execute();
- *  SEC_MMIO_WRITE_INCREMENT(kOtbnSecMMioExecute);
+ *  SEC_MMIO_WRITE_INCREMENT(kOtbnSecMmioExecute);
  * ```
  */
 enum {
-  kOtbnSecMmioExecute = 2,
+  kOtbnSecMmioExecute = 1,
```

Comment:
For my understanding, this now matches the singular sec_mmio write in otbn_execute()'s implementation, right?

https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/lib/drivers/otbn.c;l=170;drc=b9d0fb5c3cbaaf848c10f06c870a5cd0cb304599

Reply:
Correct!